tests/chainer_tests/functions_tests/pooling_tests/test_average_pooling_nd.py

```diff
@@ -52,19 +53,29 @@ def check_forward(self, x_data, use_cudnn='always'):
         pad = self.pad
         x = chainer.Variable(x_data)
         with chainer.using_config('use_cudnn', use_cudnn):
-            y = functions.average_pooling_nd(x, ksize, stride, pad)
+            y = functions.average_pooling_nd(
+                x, ksize, stride, pad, self.pad_value)
         self.assertEqual(y.data.dtype, self.dtype)
         y_data = cuda.to_cpu(y.data)
 
+        def idx_size(idx):
```

Comment:
Isn't it better to name it `patch_size`?

Reply:
`idx` and `patch` are used interchangeably in this context and neither is accurate.
I renamed to `denom`.